_posts/2020-08-14-iati-status.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: A status dashboard for IATI services
+comments: true
+author: andy
+excerpt: Welcome to Code for IATI! Learn more about what we're trying to do and how you can get involved.
+---
+
+Sometimes, web services break. They stop working. For all sorts of reasons.
+
+A server fails; Some piece of data gets corrupted; something unexpected causes a blockage in [the tubes](https://en.wikipedia.org/wiki/Series_of_tubes).
+
+These incidents are unpredictable, and can cause disruption for users. So it’s common for online services to provide a status dashboard, to automatically alert users of current issues.
+
+[Facebook has one](https://developers.facebook.com/status/dashboard/). [Twitter has one](https://api.twitterstat.us/). [Dropbox has one](https://status.dropbox.com/). [Github has one](https://www.githubstatus.com/). In fact: check it yourself. Think of an online service, and then do a search for their status dashboard. Chances are, you’ll probably find one.
+
+IATI provides a number of services, but there’s no central status dashboard. So we had a go at making one.
+
+It’s here:
+[https://status.codeforiati.org/](https://status.codeforiati.org/)
+
+It’s based on [open source code](https://github.com/IATI/IATI-Website-Tests) created by the IATI secretariat. This code runs automated tests on various IATI services, to check they are performing as expected. For example:
```

Comment:
suggest: "created by the IATI Secretariat's technical team a number of years ago"

Reply:
Yes good point. Possibly worth linking to this: https://github.com/IATI/IATI-Website-Tests/graphs/contributors

Reply:
Fixed!